src/hooks/useIsHostIn/useIsHosetIn.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import EventEmitter from 'events';
+import useDominantSpeaker from '../useDominantSpeaker/useDominantSpeaker';
+import useIsHosetIn from './useIsHostIn';
+import useVideoContext from '../useVideoContext/useVideoContext';
+import { NOTIFICATION_MESSAGE } from '../../utils/displayStrings';
+
+jest.mock('../useVideoContext/useVideoContext');
+jest.mock('../useDominantSpeaker/useDominantSpeaker');
+
+const onmsg = jest.fn(() => {});
+
+const mockedVideoContext = useVideoContext as jest.Mock<any>;
+
+describe('the useIsHosetIn hook', () => {
+  let mockRoom: any;
+
+  beforeEach(() => {
+    mockRoom = new EventEmitter();
+    mockedVideoContext.mockImplementation(() => ({
+      room: mockRoom,
+    }));
+  });
+
+  it('should return true by default', () => {
+    const { result } = renderHook(useIsHosetIn(onmsg));
+    expect(result.current).toEqual(true);
+  });
+
+  it('should return false when "participantConnected" is not the host', async () => {
+    const { result } = renderHook(useIsHosetIn(onmsg));
+    act(() => {
+      mockRoom.emit('participantConnected', 'newParticipant@HearingOfficer');
+    });
+    expect(result.current).toEqual(false);
+    expect(onmsg).toBeCalledWith(NOTIFICATION_MESSAGE.WAITING_FOR_REPORTER);
```

Comment:
you can remove the onMsg check here and from all the tests because the menubar will deal with it